tests/bluetooth/tester/src/btp/btp_bas.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef BTP_BAS_H_
+#define BTP_BAS_H_
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* BTP BAS Service Opcodes */
+#define BTP_BAS_SET_BATTERY_LEVEL             0x02
+#define BTP_BAS_BLS_SET_BATTERY_PRESENT       0x03
+#define BTP_BAS_BLS_SET_WIRED_POWER_SOURCE    0x04
+#define BTP_BAS_BLS_SET_WIRELESS_POWER_SOURCE 0x05
+#define BTP_BAS_BLS_SET_BATTERY_CHARGE_STATE  0x06
+#define BTP_BAS_BLS_SET_BATTERY_CHARGE_LEVEL  0x07
+#define BTP_BAS_BLS_SET_BATTERY_CHARGE_TYPE   0x08
+#define BTP_BAS_BLS_SET_CHARGING_FAULT_REASON 0x09
+#define BTP_BAS_BLS_SET_IDENTIFIER            0x0A
+#define BTP_BAS_BLS_SET_SERVICE_REQUIRED      0x0B
+#define BTP_BAS_BLS_SET_BATTERY_FAULT         0x0C
+
+/* Command structures */
+struct btp_bas_set_battery_level_cmd {
+	bt_addr_le_t address;
+	uint8_t level;
+} __packed;
```

Comment:
Why is `address` a parameter in these? 

It's for a GATT server device, right? Is there any use case for setting a specific value for specific remote devices?

Reply:
yeah, this feels like it should be global value

BTW I'll do BTP review in next few days (currently I'm preempted with other non-zephyr work)